src/ticket.c

```diff
@@ -434,17 +470,6 @@ void disown_ticket(struct ticket_config *tk)
 	get_time(&tk->term_expires);
 }
 
-int disown_if_expired(struct ticket_config *tk)
-{
-	if (is_past(&tk->term_expires) ||
-			!tk->leader) {
-		disown_ticket(tk);
-		return 1;
-	}
-
-	return 0;
-}
-
```

Comment:
This is unused code removed. Normally, you keep the unrelated changes separate.

Reply:
sounds good to me, sorry for the confusion!

Reply:
It is often tough to restrain oneself when one runs into something that can obviously and immediately be fixed. I do that often myself and then go back reverting the change :)